ecosystem/typescript/sdk_v2/src/transactions/authenticator/account.ts

```diff
@@ -82,3 +83,34 @@ export class AccountAuthenticatorMultiEd25519 extends AccountAuthenticator {
     return new AccountAuthenticatorMultiEd25519(public_key, signature);
   }
 }
+
+export class AccountAuthenticatorSecp256k1 extends AccountAuthenticator {
```

Comment:
need to add to the main `AccountAuthenticator` class https://github.com/aptos-labs/aptos-core/blob/main/ecosystem/typescript/sdk_v2/src/transactions/authenticator/account.ts#L11